lib/cog/startup.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Cog.Startup do
+  require Logger
+
+  @moduledoc """
+  Tasks that should happen after the main Cog supervisor has started up.
+  Note: Don't start any long running or supervised processes here. This
+  module is purely for transient book keeping/setup functions.
+  """
```

Comment:
Yeah, I'll just drop the note. I was trying to say not to start up any supervised processes here because I thought we might end up chasing down some phantom process. But in hindsight I guess that's kinda obvious.